src/test/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTaskTest.java

```diff
@@ -273,6 +275,81 @@ public void testVersion() {
     assertTrue(task.version().matches("^(\\d+\\.){2}?(\\*|\\d+)(-.*)?$"));
   }
 
+  @Test
+  public void testConvertTopicToDataStream_allowDashes() {
```

Comment:
```suggestion
  public void testConvertTopicToDataStreamAllowsDashes() {
```
dont need underscore anywhere